DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js

```diff
@@ -13,166 +13,234 @@
+
+  data.addRow([DSACampaign.name, DSACampaign.impressions,
+    DSACampaign.clicks, DSACampaign.cost]);
+
+  const options = {
+    chart: {
+      title: DSACampaign.name + ' ' + 'DSA Campaign Metrics',
+      subtitle: 'Impressions, Clicks, and Cost',
+    },
+    bars: 'horizontal', // Required for Material Bar Charts.
+  };
+
+  const chart = new google.charts.Bar(document.getElementById('bar-chart' +
+    chartNumber));
+  chart.draw(data, google.charts.Bar.convertOptions(options));
+  console.log('Drew bar graph.');
+
+  drawDSACampaignTable(DSACampaign, chartNumber);
+}
+
+function drawDSACampaignTable(DSACampaign, chartNumber) {
+  const data = new google.visualization.DataTable();
+  const data2 = new google.visualization.DataTable();
+  data.addColumn('string', 'DSA Campaign');
+  data.addColumn('string', 'Start Date');
+  data.addColumn('string', 'End Date');
+  data.addColumn('number', 'Manual CPC');
+  data.addColumn('number', 'Daily Budget');
+  data.addColumn('string', 'Location');
+  data.addColumn('string', 'Domain');
+
+
+  data2.addColumn('string', 'Target');
+  data2.addColumn('string', 'Ad Text');
+  data2.addColumn('number', 'Impressions');
+  data2.addColumn('number', 'Clicks');
+  data2.addColumn('number', 'Cost (USD)');
 
-    var data = new google.visualization.DataTable();
-    data.addColumn('string', 'DSA Campaign');
-    data.addColumn('number', 'Impressions');
-    data.addColumn('number', 'Clicks');
-    data.addColumn('number', 'Cost (USD)');
+  data.addRow([DSACampaign.name, DSACampaign.startDate, DSACampaign.endDate,
+    DSACampaign.manualCPC, DSACampaign.dailyBudget, DSACampaign.locations,
+    DSACampaign.domain]);
 
-    data.addRow([DSACampaign.name, DSACampaign.impressions, DSACampaign.clicks, DSACampaign.cost]);
+  data2.addRow([DSACampaign.targets, DSACampaign.adText,
+    DSACampaign.impressions, DSACampaign.clicks, DSACampaign.cost]);
 
-    var options = {
-        chart: {
-            title: DSACampaign.name + ' ' + 'DSA Campaign Metrics',
-            subtitle: 'Impressions, Clicks, and Cost',
-        },
-        bars: 'horizontal' // Required for Material Bar Charts.
-    };
+  const table = new google.visualization.Table(document.getElementById('table' +
+    chartNumber));
 
-    var chart = new google.charts.Bar(document.getElementById('bar-chart' + chartNumber));
-    chart.draw(data, google.charts.Bar.convertOptions(options));
-    console.log("Drew bar graph.");
+  table.draw(data, {showRowNumber: false, width: '100%', height: '50%'});
 
-    drawDSACampaignTable(DSACampaign, chartNumber);
+  const table2 = new google.visualization.Table(document.getElementById(
+      'secondtable' + chartNumber));
+
+  table2.draw(data2, {showRowNumber: false, width: '100%', height: '100%'});
+
+  createThirdRow(DSACampaign, chartNumber);
+
+  console.log('Drew table.');
 }
 
-function drawDSACampaignTable(DSACampaign, chartNumber) {
+function createThirdRow(DSACampaign, chartNumber) {
```

Comment:
Change the function name? I don't quite understand the name.
And can I see a screenshot of how the page should look.

Reply:
Done.